v21.2/jsonb.md

```diff
@@ -47,7 +43,7 @@ Examples:
 
 ## Size
 
-The size of a `JSONB` value is variable, but it's recommended to keep values under 1 MB to ensure performance. Above that threshold, [write amplification](https://en.wikipedia.org/wiki/Write_amplification) and other considerations may cause significant performance degradation.
+The size of a `JSONB` value is variable, but we recommend that you keep values under 1 MB to ensure satisfactory performance. Above that threshold, [write amplification](https://en.wikipedia.org/wiki/Write_amplification) and other considerations may cause significant performance degradation.
```

Comment:
Remove the link to the Wikipedia entry on write amplification, as that article is about SSD storage. Refer to [this Slack conversation](https://cockroachlabs.slack.com/archives/C01PHNMUFLN/p1636058091052900) for more discussion on defining read/write amplification.